### PR TITLE
Added recipe for py3nvml library

### PR DESCRIPTION
**Pull request by szymon-kuklewicz**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by szymon-kuklewicz** [comment]
@conda-forge-admin, please ping team

File: recipes/py3nvml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - enum34
+    - xmltodict
```
Comment (jakirkham) [inline recipes/py3nvml/meta.yaml]: Would drop anything from here that is not needed for the build (like `enum34` and `xmltodict`).

File: recipes/py3nvml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - py3nvml
+
+about:
+  home: "https://github.com/fbcotter/py3nvml.git"
+  license: BSD
```
Comment (jakirkham) [inline recipes/py3nvml/meta.yaml]: What kind of `BSD` license?

File: recipes/py3nvml/meta.yaml (hunk 1 above)
Comment (szymon-kuklewicz) [reply recipes/py3nvml/meta.yaml]: Removed everything except pip and python as suggested

File: recipes/py3nvml/meta.yaml (hunk 2 above)
Comment (szymon-kuklewicz) [reply recipes/py3nvml/meta.yaml]: Specified detailed BSD 3-clause

**Comment by szymon-kuklewicz** [comment]
@conda-forge-admin, please ping team

**Comment by szymon-kuklewicz** [comment]
@conda-forge-admin, please ping team

File: recipes/py3nvml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/fbcotter/py3nvml/archive/{{ version }}.tar.gz"
+  sha256: "6f111b8bc460228ca5c5b430f80d0dbc27b11e8ad5a589bd0be51fae879bb75e"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```
Comment (npavlovikj) [inline recipes/py3nvml/meta.yaml]: I think this recipe can be `noarch`, please add `noarch: python`.

File: recipes/py3nvml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+    - enum34      # [py < 34]
+    - xmltodict
+
+test:
+  imports:
+    - py3nvml
+
+about:
+  home: "https://github.com/fbcotter/py3nvml.git"
```
Comment (npavlovikj) [inline recipes/py3nvml/meta.yaml]: You can remove `.git` from the url.

File: recipes/py3nvml/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: "LICENSE"
+  summary: "Python 3 Bindings for the NVIDIA Management Library"
+  doc_url: "https://py3nvml.readthedocs.io/en/latest/"
+  dev_url: "https://github.com/fbcotter/py3nvml.git"
```
Comment (npavlovikj) [inline recipes/py3nvml/meta.yaml]: Please remove `.git` from dev_url.

File: recipes/py3nvml/meta.yaml (hunk 3 above)
Comment (chrisburr) [reply recipes/py3nvml/meta.yaml]: In this case it can't due to: `- enum34      # [py < 34]`

**Comment by npavlovikj** [comment]
Ah, I was too late :)

**Pull request merged**